make/compiler/Makefile.pgi

```diff
@@ -51,6 +51,8 @@ GEN_DYNAMIC_FLAG = -Bdynamic
 LIB_STATIC_FLAG =
 LIB_DYNAMIC_FLAG = -shared
 
+# TODO: set target architecture for optimization
+
```

Comment:
I think it's nice to use the explicit flag when optimization is on, since people might  (wrongly) suspect their C compiler always uses IEEE floats unless they give a flag (since that's what GCC does).